ring/ring_http.go

```diff
@@ -36,15 +36,17 @@ type httpResponse struct {
 }
 
 type ingesterDesc struct {
-	ID                  string    `json:"id"`
-	State               string    `json:"state"`
-	Address             string    `json:"address"`
-	HeartbeatTimestamp  time.Time `json:"timestamp"`
-	RegisteredTimestamp time.Time `json:"registered_timestamp"`
-	Zone                string    `json:"zone"`
-	Tokens              []uint32  `json:"tokens"`
-	NumTokens           int       `json:"-"`
-	Ownership           float64   `json:"-"`
+	ID                       string    `json:"id"`
+	State                    string    `json:"state"`
+	Address                  string    `json:"address"`
+	HeartbeatTimestamp       time.Time `json:"timestamp"`
+	RegisteredTimestamp      time.Time `json:"registered_timestamp"`
+	ReadOnly                 bool      `json:"readonly"`
+	ReadOnlyUpdatedTimestamp time.Time `json:"readonly_updated_timestamp"`
```

Comment:
[nit, non blocking] Have you considered `read_only*` (separated by `_`) to keep it consistent with the protobuf message definition?